.gitignore

```diff
@@ -6,6 +6,7 @@
 /ports/android/libs
 /ports/android/local.properties
 /ports/android/obj
+/ports/geckolib/target
```

Comment:
I got some files there when I was building geckolib, or so I _thought_ - rebuilding now doesn't do it. I think it must have been some background compilation done in Atom while I was editing the files. Reverting that part now, thanks for seeing it.